src/display/canvas.js

```diff
@@ -639,8 +599,24 @@ class CanvasExtraState {
     this.updatePathMinMax(transform, box[2], box[3]);
   }
 
-  getPathBoundingBox() {
-    return [this.minX, this.minY, this.maxX, this.maxY];
+  getPathBoundingBox(pathType = PathType.FILL, transform = null) {
+    const box = [this.minX, this.minY, this.maxX, this.maxY];
+    if (pathType === PathType.STROKE) {
+      if (!transform) {
+        throw new Error("Stroke bounding box must include transform.");
```

Comment:
Perhaps just use `unreachable` instead, rather than throwing manually, since I'm assuming that this code-path shouldn't be hit unless an implementation error was made.